tools/internal_ci/helper_scripts/prepare_build_windows.bat

```diff
@@ -39,6 +39,9 @@ set PATH=%LOCALAPPDATA%\Microsoft\dotnet;%PATH%
 @rem TODO(jtattermusch): try to eliminate the dependency on Go
 choco install golang -y --version 1.13.1 --limit-output
 
+@rem Install Python 3.8.0
+powershell -File tools\internal_ci\helper_scripts\install_python38.ps1
```

Comment:
if there order of installing golang and python38 matters (it sounds like it does, we have to add a comment).

Thinking forward, a while ago I've split the kokoro jobs into language specific jobs so that with some extra work the build scripts become a bit more selfcontained - the python jobs could only install python dependencies and C# jobs could only install C# dependencies. If we figure out a good way of doing that, it could make our build more stable by installing fewer dependencies and reducing the chance that one languages will be broken due to dependency changes introduced by another language.

Reply:
I added a comment. It's uncertain if installation order matters until we have enough successful builds. I'm somehow confident about other two changes will reduce the flakiness.

In one of the versions, I tried to install Python 3.8 on demand. However, I found the artifact / package building jobs are all-languages-in-one. 4/6 of the batch scripts in Windows CI needs Python 3.8. So, I thought it would be simpler to install for all of them.

I agree that language-specific jobs are ideal and easier to maintain :)